src/lsp/methods/suggestEnvironmentVars.ts

```diff
@@ -1,8 +1,14 @@
 import * as vscode from "vscode";
-import { execSync } from "child_process";
+import { ChildProcess, execSync } from "child_process";
 import * as path from "path";
 import * as fs from "fs";
-import triggers from "./triggers";
+import triggers from "../../triggers";
+import {
+  ErrorCodes,
+  IJSONRPCResponse,
+  logToChannel,
+} from "../response/generalResponse";
```

Comment:
We can get imports based on project root everywhere if possible.

Reply:
Will do in the next MR, tried, taking a lot of time